docs/resources/object_bucket.md

```diff
@@ -40,6 +119,27 @@ The `CORS` object supports the following:
 * `expose_headers` (Optional) Specifies expose header in the response.
 * `max_age_seconds` (Optional) Specifies time in seconds that browser can cache the response for a preflight request.
 
+The `lifecycle_rule` (Optional) object supports the following:
+
+* `id` - (Optional) Unique identifier for the rule. Must be less than or equal to 255 characters in length.
```

Comment:
Could you add an example without id? Because it is present everytime it gives the impression that it is mandatory.

Reply:
I've just added a new  test and example in the documentation